libs/ledger-live-common/src/families/aptos/js-getFeesForTransaction.ts

```diff
@@ -50,6 +50,14 @@ export const getFee = async (
   if (account.xpub) {
     try {
       const publicKeyEd = new Ed25519PublicKey(account.xpub as string);
+
```

Comment:
Can `transaction.firstEmulation` be anything other than a boolean?

Wouldn't `(transaction.useAllAmount && transaction.firstEmulation)` suffice?